internal/worker/worker.go

```diff
@@ -4,14 +4,17 @@ import (
 	"context"
 	_ "embed"
 	"fmt"
-	"github.com/mprimi/go-bench-away/internal/core"
-	"golang.org/x/sys/unix"
 	"io"
+	"log"
```

Comment:
I don't think the rest of this project uses `log`

Reply:
Will change to `fmt`.